<?xml version="1.0"?>
<launch>
  <include file="$(find multi_turtlebot3_navigation)/launch/move_base_namespace0.launch">
  <arg name="robot_namespace" value="tb3_0" />
  </include>

  <include file="$(find multi_turtlebot3_navigation)/launch/move_base_namespace1.launch">
  <arg name="robot_namespace" value="tb3_1" />
  </include>

  <include file="$(find multi_turtlebot3_navigation)/launch/move_base_namespace2.launch">
  <arg name="robot_namespace" value="tb3_2" />
  </include>
</launch>	